README.md

```diff
@@ -537,10 +537,10 @@ You can also stake through Binance and Huobi.
 
 ### Basic visual representation
 
-![Architecture](./graphs/architecture-pretty.png)
+![Architecture](https://github.com/AdExNetwork/adex-protocol/blob/master/graphs/architecture-pretty.png)
 
 * The box-shaped platform and AdView are client-side software
-* Round-shaped items represent parts of the AdEx peer-to-peer network (in practice, [many validators and markets may exist](/graphs/real-world.svg))
+* Round-shaped items represent parts of the AdEx peer-to-peer network (in practice, [many validators and markets may exist](https://github.com/AdExNetwork/adex-protocol/blob/master/graphs/real-world.svg))
```

Comment:
same here. Either use https://raw.githubusercontent.com/AdExNetwork/adex-protocol/master/graphs/real-world.svg
or leave it as it is

Reply:
fixed